smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/fault/KafkaFailureHandler.java

```diff
@@ -29,4 +29,8 @@ public static Strategy from(String s) {
 
     <K, V> CompletionStage<Void> handle(IncomingKafkaRecord<K, V> record, Throwable reason);
 
+    default void terminate() {
```

Comment:
As the default is do nothing is this needed?

Or was it mostly to make explicit that this case doesn't require anything?

Reply:
The DLQ implementation overrides it, and close the producer.
Other strategies don't need to do anything.